Makefile

```diff
@@ -27,7 +27,7 @@ lint:
 	tox -elint
 
 lint-roll:
-	isort --recursive pytest_asyncio_networking_simulator tests
+	isort --recursive pytest_asyncio_network_simulator tests
```

Comment:
This module name is a real mouthful. Maybe since it is explicitly for pytest, the simulator part is strongly enough implied? Then the package and module could be `pytest_asyncio_network`.